css/media.css

```diff
@@ -0,0 +1,413 @@
+  }
+
+  .header {
+    max-width: 90%;
+    margin: 0 auto;
+  }
+}
+
+@media screen and (max-width: 900px) {
+
+  .header {
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 5px;
+  }
+
+  .header__logo {
+    display: none;
+  }
+
+  .header__logo-mobile {
+    display: inline-block;
+    margin-top: 23px;
+  }
+
+  .navbar {
+    display: none;
+  }
+
+  .burger {
+    padding: 0;
+    border: none;
+    cursor: pointer;
+    position: relative;
+    display: inline-block;
+    width: 61px;
+    height: 9px;
+    margin-top: 5px;
+    background-color: #000;
+  }
+
+  .burger::after {
+    content: '';    
+    position: absolute;
+    left: 0;
+    margin-top: 12px;
+    display: inline-block;
+    width: 61px;
+    height: 9px;
+    background-color: #000;
+  }
+
+}
+
+
+
+@media screen and (max-width: 768px) {
+
+  .container {
+    max-width: 100vw;
+  }
+  
+   .market-section {
+    padding-top: 16px;
+    padding-bottom: 100px;
+  }
+  
+  .banner {
+    padding: 81px 0 0 43px;
+    min-height: 416px;
+    background-position: 17%;
+  }
+  
+  .banner__title {
+    font-size: 50px;
+    line-height: 48px;
+    max-width: 516px;
+    margin-bottom: 43px;
+  }
+
+  .banner__link-more {
+    font-size: 20px;
+  }
+  
+  .banner__link-more::after { /* Разобраться почему не управляется ширина */
+    margin-left: 13px;
+  }
+  
+  /* ---------------------------------------------- */
+  
+  .section__subtitle {
+    font-size: 30px;
+  }
+
+  .market-section__subtitle {
+    margin: 62px 18px 28px 0;
+  }
+
+
+  .products {
+    justify-content: space-around;
+  }
+
+  .product-card {
+    width: 280px;
+    padding: 0 0 0 34px;
+  }
+
+  .products__product-card:nth-child(2) {
+    padding: 0 0;
+    margin: 0 8px 0 0;
+  }
+
+  .products__product-card:nth-last-child(-n+2) {
+    display: none;
+  }
+  
+  .product-card__picture {
+    margin: 0 0 28px 13px;
+  }
+
+  .product-card__picture img {
+    width: 234px;
+    height: 219px;
+  }
+  
+  .product-card__name {
+    font-size: 20px;
+    margin: 0 0 13px 0;
+  }
+
+  .product-card__price {
+    margin-right: 58px;
+    font-size: 25px;
+  }
+
+  .product-card::after {
+    display: none;
+  }
+
+  .circles {
+    margin-right: 11px;
+  }
+
+
+  .circles__link {
+    width: 20px;
+    height: 20px;
+    margin-left: 6px;
+  }
+
+  .color-option {
+    padding: 0 0 0 0;
+  }
+}
+
+@media screen and (max-width: 620px) {
+
+  .products__product-card:nth-child(2) {
+    display: none;
+  }
+}
+
+@media screen and (max-width: 375px) {
+
+  .container {
+    max-width: 375px;
+  }
+
+  .header {
+    min-height: 53px;
+    padding: 0 28px 0 30px;
+  }
+
+  .header__logo-mobile {
+    margin-top: 20px;
+  }
+
+  .header__logo-mobile img {
+    width: 104px;
+  }
+
+  .burger {
+    width: 41px;
+    height: 5px;
+    margin-top: 9px;
+  }
+
+  .burger::after {
+    margin-top: 8px;
+    width: 41px;
+    height: 6px;
+  }
+
+  
+   /* ------------------------- */
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Axion2/pullRequest?prid=5654146)